keys_test.go

```diff
@@ -35,14 +37,16 @@ func TestKeys(t *testing.T) {
 	assert.NotEmpty(t, keys)
 
 	key = NewKey{
-		Name:       name + "-changed",
-		Capability: map[string][]string{"b": {"publish"}},
+		Name:            name + "-changed",
+		Capability:      map[string][]string{"b": {"publish"}},
+		RevocableTokens: false,
 	}
 
 	k, err = client.UpdateKey(app.ID, k.ID, &key)
 	assert.NoError(t, err)
 	assert.Equal(t, key.Name, k.Name)
 	assert.Equal(t, key.Capability, k.Capability)
+	assert.Equal(t, key.RevocableTokens, k.RevocableTokens)
```

Comment:
I think this is probably out of scope for this change but comparing each individual parameter in the struct is not necessary, at least with the `testify/assert` package, because you can directly compare two structs and it'll show you a diff where the parameters differ

Reply:
Sounds good. I'll make a note and implement this at some point.